CONTRIBUTING.md

```diff
@@ -69,6 +69,7 @@ below:
 * Mark Worsfold (Met Office, UK)
 * Shubhendra Singh Chauhan (DeepSource, India)
 * Ying Zhao (Bureau of Meteorology, Australia)
+* Anja Schubert (Bureau of Meteorology, Australia)
```

Comment:
I think this list is supposed to be in last-name-alphabetical order.

Reply:
Fixed